api4/resolver_channel_test.go

```diff
@@ -436,7 +436,7 @@ func TestGetPrettyDNForUsers(t *testing.T) {
 				LastName:  "last2",
 			},
 		}
-		assert.Equal(t, "nick2", getPrettyDNForUsers("nickname_full_name", users, "user1"))
+		assert.Equal(t, "nick2", getPrettyDNForUsers("nickname_full_name", users, "user1", map[string]string{}))
```

Comment:
Do we need / want tests to check the cache is working as expected?

Reply:
Don't think that's necessary.

Reply:
Just to be sure this will work in the future, I'll add a small check.